### PR TITLE
Add "FLPSUITE_LATEST" variable to ALIDIST_SLUG for daily tags

### PR DESCRIPTION
**Pull request by TimoWilken**
The FLP suite needs special handling, so add a variable (which can be used when scheduling a build in Jenkins), which is substituted for the current FLP suite version.

### CODE REVIEW

**Review by davidrohr — APPROVED** [review]

File: daily-tags.sh [hunk 1]
```diff
@@ -3,6 +3,22 @@ set -ex
 
 # Check for required variables
 ALIDIST_SLUG=${ALIDIST_SLUG:-alisw/alidist@master}
+if echo "$ALIDIST_SLUG" | grep -q '!!FLPSUITE_LATEST!!'; then
```
Comment (ktf) [inline daily-tags.sh:6]: I would just use FLPSUITE_LATEST. Why the extra `!!`?

Comment (TimoWilken) [reply daily-tags.sh:6]: To make it obvious that it'll be replaced and isn't just a normal tag.

Comment (ktf) [reply daily-tags.sh:6]: Well, it would have been good enough. Anyways, fine. Merging this. Can you update the jenkins job accordingly?

Comment (ktf) [reply daily-tags.sh:6]: (I mean DailyO2Dataflow)

**Pull request merged**